src/main/java/com/servicewizard/config/TransformerType.java

```diff
@@ -0,0 +1,16 @@
+package com.servicewizard.config;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum TransformerType {
+	CUSTOM, ANGULAR, DEFAULT_HTML, DEFAULT_MARKDOWN;
```

Comment:
Could we rename CUSTOM to CUSTOM_TEMPLATE? The reason is I think we should (eventually) provide two ways for users to customize the output - by providing a template, and by providing a custom Transformer class. There would be a CUSTOM_TRANSFORMER type added (or something) later on.